xbmc/addons/AddonBindings.cmake

```diff
@@ -5,7 +5,6 @@
 # Keep this in alphabetical order
 set(CORE_ADDON_BINDINGS_FILES
     ${CORE_SOURCE_DIR}/xbmc/cores/AudioEngine/Utils/AEChannelData.h
-    ${CORE_SOURCE_DIR}/xbmc/filesystem/IFileTypes.h
```

Comment:
As note, to remove here was a bad idea from me. This header also be used currently by PVR on `libXBMC_addon.h`.

This line change becomes removed.